modules/conversion-center/assets/scss/widgets/link-in-bio-base.scss

```
@@ -139,10 +145,15 @@
 	}
 
 	&__identity {
+		display: grid;
+		grid-template-columns: 1fr;
+		grid-template-rows: auto;
 		margin-block-end: 14px;
+		margin-block-start: calc(var(--e-link-in-bio-gutter-block-start) * -1);
 
 		.e-link-in-bio__identity-image {
```

A general comment: it would be great if we can reduce the length of the variables, without losing meaning. I am thinking e-link-in-bio can be reduced to be.

```suggestion
		.eb-identity-image {
```

---

@louiswol94 also as above, putting this on your radar